### ISSUE TITLE
Adding mbed id definition for Atmel boards

### ISSUE THREAD

**Issue by mvolstad** [ARMmbed/mbed-ls#40]
Atmel boards are missing from the manufacture_ids dictionary in lstools_base.py.

**Comment by mvolstad** [comment]
Prepared a fix in fork:
https://github.com/mvolstad/mbed-ls/commit/9e76cdb70c31e4d027cd142988e1740bb8893ab3

**Comment by ciarmcom** [comment]
ARM Internal Ref: IOTSYST-776

**Comment by PrzemekWirkus** [comment]
Released with v0.1.28:
```
Submitting dist\mbed-ls-0.1.28.zip to https://pypi.python.org/pypi
Server response (200): OK
```
